File: Class_Object.cpp
#include <bits/stdc++.h>
using namespace std;

// class 
class Student{
    public:
    string name;
    int roll;


    // contractor
    Student(string name,int roll)
    {
        this->name = name;
        this->roll= roll;
    }
};

int main()
{
    Student yeasin("yeasin", 10);
    cout<<yeasin.name;
    return 0;
}

File: Class_Object.py
# class
class Student:
    name = ''
    roll = 0

    def __init__(self, name, roll) -> None:
        self.name = name
        self.roll = roll

# object
yeasin = Student('yeasin', 10)

print(yeasin.name)
print(yeasin.roll)